scenarios/test_scenarios/gs_forced_collision_test_vehicle_rear_brake.osm

```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Are these the same files? @zhengc84 
`gs_forced_collision_test_vehicle_rear_brake.osm` and `....break.osm`